uptrain/operators/language/llm.py

```diff
@@ -70,9 +74,12 @@ async def async_process_payload(
                             openai.error.APIError,
                             openai.error.Timeout,
                             openai.error.TryAgain,
+                            cohere.error.CohereAPIError,
```

Comment:
litellm standardizes exceptions to the openai format, so you don't need to check cohere errors https://docs.litellm.ai/docs/exception_mapping

Reply:
Made the change. Thanks!